src/components/molecules/Avatar/component.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC, useCallback, useEffect, useState } from 'react';
+import { Props } from './props';
+import styled from '@emotion/styled';
+
+export const Avatar: FC<Props> = ({ url, alt, ...rest }: Props) => {
+  return(
+    <div {...rest}>
+      <img src={url} alt={alt}/>
+    </div>
+  )
+}
+
+export const AvatarStyle = styled(Avatar)`
+    padding: 0px;
+    margin: 0px;
+    border-radius: 50%;
```

Comment:
Also you have to overflow: hidden to make children not be visible outside parent element